packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx

```diff
@@ -47,10 +47,12 @@ function SaveMnemonic({ className, onNextStep, onPreviousStep, seed }: Props): R
             />
           ))}
         </div>
-        <CopyToClipboard text={seed}>
+        <CopyToClipboard
+          onCopy={_onCopy}
```

Comment:
callback signature is `onCopy(text, result)` where result is a boolean indicating is copying was successful. Not sure when can it fails, but perhaps it's worth handling?

Reply:
Missed that, thanks! Will fix.